components/sections/ProjectGroupbyStatus.tsx

```diff
@@ -109,6 +109,7 @@ export default function ProjectGroupbyStatus(props: Props) {
         return (
           <IssuesTable
             issues={getIssues(group, rowState.stageType, rowState.stageKey)}
+            showTargetDate={true}
```

Comment:
Not important, but just as an FYI, React also lets you just pass `showTargetDate` for boolean props instead of having to do `showTargetDate={true}`, e.g. `<IssuesTable issues={} showTargetDate />`.